diff --git a/.env b/.env
index 399a487..b5173a3 100644
--- a/.env
+++ b/.env
@@ -9,21 +9,21 @@ ODOO_IMAGE=adhoc/odoo-adhoc
 ODOO_MINOR=17.0
 
 # debe ser [CualquierCosa].odoo.localhost
-DOMAIN=17.odoo.localhost
+DOMAIN=core.odoo.localhost
 
 # Must be a number between 0 and 254 (for now used mainly for dns network)
 # TODO check if we want to re-use for domain, odoo_minor or other
 # TODO maybe rename, the name confuse the internal env var on odoo container ODOO_VERSION (that has dot version number like 15.0)
-ODOO_VERSION=17
+ODOO_VERSION=30
 
 # Descomentar si queremos que se filtre la base de datos por el suddominio
-# DBFILTER='%d.*$'
+DBFILTER='%d.*$'
 
 # comentar o modificar segun corresponda
-SERVER_WIDE_MODULES=base,web,server_mode,saas_client
+# SERVER_WIDE_MODULES=base,web,server_mode,saas_client
 
 # descomentar si no se quieren tener en cuenta los src/repositories (repositorios dentro de la imagen)
-# IGNORE_SRC_REPOSITORIES=True
+IGNORE_SRC_REPOSITORIES=True
 
 #####################################
 # PARAMETROS QUE NO SE SUELEN CAMBIAR
